tests/elster_xml/common/test_basic_xml_data_representation.py

```diff
@@ -0,0 +1,83 @@
+from xml.etree import ElementTree
+
+import pytest
+
+from erica.elster_xml.common.basic_xml_data_representation import EEmpfaenger, ENutzdatenHeader, ENutzdaten, \
+    ENutzdatenblock, EDatenTeil, EElster, EXml, construct_basic_xml_data_representation
+from erica.elster_xml.common.xml_conversion import convert_object_to_xml
+
+
+@pytest.fixture
+def nutzdaten():
+    return ENutzdaten()
+
+
+@pytest.fixture
+def default_function_args(nutzdaten):
+    return "ID", "TEXT", nutzdaten, "VERSION", "TICKET"
+
+
+class TestEEmpfaenger:
+    def test_attributes_set_correctly(self):
+        resulting_empfaenger = EEmpfaenger("ID", "TEXT")
+        assert resulting_empfaenger.xml_attr_id == "ID"
+        assert resulting_empfaenger.xml_only_text == "TEXT"
+        assert len(vars(resulting_empfaenger)) == 2
+
+
+class TestENutzdatenHeader:
+    def test_attributes_set_correctly(self):
+        resulting_nutzdaten_header = ENutzdatenHeader("ID", "TEXT", "VERSION", "TICKET")
+        assert resulting_nutzdaten_header.Empfaenger == EEmpfaenger("ID", "TEXT")
+        assert resulting_nutzdaten_header.xml_attr_version == "VERSION"
+        assert resulting_nutzdaten_header.NutzdatenTicket == "TICKET"
+        assert len(vars(resulting_nutzdaten_header)) == 3
+
+
+class TestENutzdatenblock:
+    def test_attributes_set_correctly(self, nutzdaten, default_function_args):
+        resulting_nutzdaten_block = ENutzdatenblock(*default_function_args)
+        assert resulting_nutzdaten_block.NutzdatenHeader == ENutzdatenHeader("ID", "TEXT", "VERSION", "TICKET")
+        assert resulting_nutzdaten_block.Nutzdaten == nutzdaten
+        assert len(vars(resulting_nutzdaten_block)) == 2
+
+
+class TestEDatenTeil:
+    def test_attributes_set_correctly(self, default_function_args):
```

Comment:
Because there is no if. There's just input. Therefore I don't need an if. I *really* tried my best in naming the tests to the convention and even inserted empty lines (!!!!! 😂 ) - here it just does not make sense and I refuse to add it. 😂